itou/templates/siaes/edit_siae_summary.html

```diff
@@ -0,0 +1,95 @@
+{% extends "layout/content_small.html" %}
+{% load format_filters %}
+{% load bootstrap4 %}
+{% load theme_inclusion %}
+
+{% block title %}Modifier les coordonnées de votre structure{{ block.super }}{% endblock %}
+
+{% block breadcrumbs %}
+{% include 'siaes/edit_siae_breadcrumb.html' %}
+{% endblock %}
+
+{% block content %}
+
+<div class="card mt-6 p-2 bg-white">
+    <div class="card-body">
+        <p class="mb-0"><b>Etape 3</b>/3 : Aperçu</p>
+    </div>
+</div>
+
+<div class="card mt-6 p-2 bg-white">
+    <div class="card-body">
+        <div class="text-center">
+            <h1 class="h1-hero"><i class="ri-home-smile-fill ri-2xl"></i></h1>
+            <h1 class="h1">Aperçu de la fiche de ma structure</h1>
+        </div>
+
+        <div class="c-card shadow my-4 p-2 bg-white">
+            <div class="card-body">
+                <p><i class="ri-home-7-line mr-2"></i>{{ siae.kind }} - {{ siae.display_kind }} </p>
+                <hr>
+                <h2 class="h1">
+                    {{ siae.display_name }}
+                </h2>
+                <p> {{ siae.address_on_one_line }} </p>
+            </div>
+        </div>
+
+        <form method="post" action="" class="js-prevent-multiple-submit">
+            {% csrf_token %}
+
+            <h3>Son activité</h3>
+            <p>{{form_data.description}}</p>
```

Comment:
```suggestion
            <p>{{ form_data.description }}</p>
```
Même remarque ligne 45.

Reply:
corrigé.